ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/router-bgp.j2

```diff
@@ -1102,16 +1090,12 @@ router bgp {{ router_bgp.as }}
 {%             if neighbor.additional_paths.receive is arista.avd.defined(true) %}
       neighbor {{ neighbor.ip_address }} additional-paths receive
 {%             endif %}
-{%             if neighbor.additional_paths.send.any is arista.avd.defined(true) %}
-      neighbor {{ neighbor.ip_address }} additional-paths send any
-{%             elif neighbor.additional_paths.send.backup is arista.avd.defined(true) %}
-      neighbor {{ neighbor.ip_address }} additional-paths send backup
-{%             elif neighbor.additional_paths.send.ecmp is arista.avd.defined(true) %}
-      neighbor {{ neighbor.ip_address }} additional-paths send ecmp
-{%             elif neighbor.additional_paths.send.ecmp_limit is arista.avd.defined %}
-      neighbor {{ neighbor.ip_address }} additional-paths send ecmp limit {{ neighbor.additional_paths.send.ecmp_limit }}
-{%             elif neighbor.additional_paths.send.limit is arista.avd.defined %}
-      neighbor {{ neighbor.ip_address }} additional-paths send limit {{ neighbor.additional_paths.send.limit }}
+{%             if neighbor.additional_paths.ecmp_limit is arista.avd.defined %}
```

Comment:
this should also check if `send` has the correct value. Same for limit.

Reply:
i added valid_values key under 'send' in schema. Also added min, max value for limit.